config/sites/signage.sites.uiowa.edu/config_split.patch.core.entity_view_display.node.sign.default.yml

```
@@ -0,0 +1,17 @@
+adding:
+  dependencies:
+    config:
+      - field.field.node.sign.og_audience
+    module:
+      - field_delimiter
+      - options
+  content:
+    og_audience:
+      type: entity_reference_label
+      label: above
+      settings:
+        link: true
+      third_party_settings: {  }
+      weight: 103
+      region: first
```

```suggestion
  hidden:
    og_audience: true
```
I would set this to hidden like the other EVD configs.